il-operator/controller/env/env.go

```diff
@@ -222,11 +228,7 @@ func StateManagerNamespace() string {
 }
 
 func ArgocdNamespace() string {
```

Comment:
Can we replace `ArgocdNamespace()` calls with `CloudKnitSystemNamespace()` instead of keeping this method?

Reply:
I kept it since not sure if CloudKnitSystemNamespace() is where argocd should be deployed.